src/nvim/tui/terminfo.c

```diff
@@ -14,11 +14,13 @@
 #include "nvim/tui/terminfo.h"
 #include "nvim/tui/terminfo_defs.h"
 
+
 #ifdef __FreeBSD__
 # include "nvim/os/os.h"
 #endif
 
 #ifdef INCLUDE_GENERATED_DECLARATIONS
+# include "os/env.c.generated.h"
```

Comment:
```suggestion

```

You should get rid of this include and instead remove the header gaurd (`#ifdef __FreeBSD__` and `#endif`) around the `nvim/os/os.h` include. That should include the required `os_getenv()` (or `os_env_exists()` as I suggested in another comment) correctly.

Reply:
Thank you for your suggestions. will apply the changes and test it out